backend/app/services/embedding_service/service.py

```
@@ -1,225 +1,293 @@
+                        max_workers=1,
+                        thread_name_prefix="llm-worker",
+                    )
+        return self._llm_executor
 
-        except Exception as e:
-            logger.error(f"Error summarizing profile for {profile.github_username}: {str(e)}")
-            raise
+    @property
+    def model(self) -> SentenceTransformer:
+        if self._shutting_down:
+            raise RuntimeError("EmbeddingService is shutting down")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)